working-with-your-data/working-with-your-own-data.md

```diff
@@ -45,7 +45,7 @@ If you are retrieving your data from online, perhaps from a publicly available r
 
 ![Terminal tab](screenshots/rstudio-session-terminal.png)
 
-**Step 2)** Copy over the [wget template script](./template-script/wget-TEMPLATE.sh).
+**Step 2)** Copy over the [wget template script]({{site.repository_url}}/tree/master/working-with-your-data/template-scripts/wget-TEMPLATE.sh).
```

Comment:
Do we want them pointed to the github page or to the file itself? I can see an argument either way, and don't have a strong preference. I guess the github link gets them to see the file, rather than downloading it, which is probably better. 

We might need somewhat more detailed instructions than just "copy over".

```suggestion
**Step 2)** Copy over the [wget template script](template-scripts/wget-TEMPLATE.sh).
```

Reply:
This is just the title of the section, the instructions tell them to check mark the file and go to “More” and “Copy to”.

Reply:
If it's the title, why not make it a header?

Reply:
Didn’t seem necessary to include each step in the doctoc.

Reply:
I like the link to GitHub - I think we can talk about headers on #14.